src/deep-cache/lib/Driver/LocalStorageDriver.js

```diff
@@ -98,15 +98,20 @@ export class LocalStorageDriver extends AbstractDriver {
    * @private
    */
   _flushStale() {
-    let removedKeys = [];
+    let keysToRemove = [];
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).](https://www.codacy.com/app/MitocGroup/deep-framework/file/1763574544/issues/source?bid=2895475&fileBranchId=3063691#l101)